docs/ABOUT.md

```diff
@@ -1,5 +1,7 @@
-C# is a modern, object oriented language developed and maintained by Microsoft. The learning curve is small, reduced even if the developer has experience with other 'C-Style' languages.
+C# is a multiparadigm language with lots of features. While at its core it's still an object oriented language that's modeled after Java it supports first class functions, SQL-like syntax to access containers, automatic getters and setters, pattern matching, direct memory access in an otherwise garbage collected environment, statically typed, dynamically typed and (to a certain degree) type inferred programming and many other features in a constantly growing feature set.
```

Comment:
Maybe reword the type inference a bit to something like: "and supports basic type inference"?

Reply:
I think we should definitely add a mention of async/await in some way, for example by mentioning that C# has an elegant, simple way to handling asynchrony.

Reply:
I'm not sure automatic getters and setters should be in the list. They seem to be a relatively minor feature.